### PR TITLE
chromebooks docs improvements part 1

### PR DESCRIPTION
**Pull request by thenameisluk**
this is part one of who knows many changes

things to be done can be seen [here](https://github.com/thenameisluk/imagebuilder/blob/main/luk's%20special%20todo)
will already give you part to review and give feedback and will work on the to-do list from different branch

changes in this part
- add special to-do list for my pr's (for tracking stuff to be done next)
- gathered more data on dualboot installation (preparing to write a guide for that)
- improved luks installation guide slightly + added better screenshot's
- fixed some errors and improved in prepration guide
- split regular installation and dd one (now legacy one)
- moved installation guides into different folder (it's better to group stuff instead of having everything in one folder)
- moved change name guide from luks installation to own .md file (it can be done at any time without issues)
- reworked basic installation
- updated doc/first-steps to point into the right file again
- added wacom tablet guide from the issue with nice image
- added and improved pictures of chromebooks

again everything can be explored starting from [here](https://github.com/thenameisluk/imagebuilder/blob/main/doc/chromebooks/preparing-chromebook.md)

### CODE REVIEW

**Comment by thenameisluk** [comment]
veyron note and before are already in the repo main

not sure why github included them

**Comment by thenameisluk** [comment]
so i have all required notes to pull off dualboot
but it depends on velvet tools
https://github.com/thenameisluk/imagebuilder/blob/2nd-pr/doc/chromebooks/installation/dualboot-installation-note.txt

**Comment by hexdump0815** [comment]
i'll try to review it during the next days as i would need a bit of calm time to look though it, but just from quickly scrolling through it i can already leave a big "thank you" here for all your contributions ...

File: doc/chromebooks/installation/luks-installation.md [hunk 1]
```diff
@@ -48,6 +38,8 @@ we will export the correct drive as a environment variable but i still encourage
 
 _Note. remmber to do it whenever you want to repeat any of steps below_
```
Comment (hexdump0815) [inline doc/chromebooks/installation/luks-installation.md:40]:
```suggestion

emmc disks can usually easily identified by the fact that there are also the following two (not to be used) devices for them as well: mmcblkxboot0, mmcblkxboot1

```

File: doc/chromebooks/installation/luks-installation.md [hunk 2]
```diff
@@ -101,19 +92,20 @@ _Note. some newer chromebooks can boot 64mb partitions but for now let's stick t
...
 
 _Note. you don't need to format them or select specyfic type, just make them correct size we will format them later_
 
-5. make filesystems
+_Warning. this images are for software look showcase and to make it easier for you to chose the most comfortable one for you <3, partition your device the way described in the guide, not the way presented on the images_
+
+1. make filesystems
 
 format boot partition
 ```
```
Comment (hexdump0815) [inline doc/chromebooks/installation/luks-installation.md:111]: two lines lower there is a typo lable - label ... i'm adding the note here as the below lines are not changed, so i cannot add the note there directly easily ... i aslo spotted that typo a bit further down once more

**Review by hexdump0815 — CHANGES_REQUESTED** [review]
found some minor things here and there - also please double check the images: some of them seem to be there as jpeg and png and it would be good to only include the referenced version or make sure only one gets referenced and kept in the tree if possible

in general: once more a big thank you for those contributions - i think with this documentation it should be much easier for others to follow the chromebook installation and customization process

**Comment by thenameisluk** [comment]
@hexdump0815 i've only used jpeg's in docs because of the size benefits (it would be faster to load one via bad internet) but i kept png's incase some would want to use a better quality version of these images (also the jpeg's are half the resolution of png to make them even smaller)

**Comment by hexdump0815** [comment]
@thenameisluk - i would prefer to only include the small jpeg files then as otherwise the repo will grow quite a bit in size due to the images ... mid- or long-term we might think about creating a separate imagebuilder-doc or velvet-os-doc repo to hold the doc part and keep the old repo for the code and maybe a simple readme per system to keep that small and that approach would also give more room for more documentation, images etc. without affecting the size of the code-base ... i started something similar for firmware files some time ago already: https://github.com/hexdump0815/imagebuilder-firmware (still to be improved and some more files from the current tree will go there over time as well i think)

File: doc/chromebooks/preparing-chromebook.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Intro ... getting stuff to boot
```
[the remark this answers is not in the document]
Comment (thenameisluk) [reply doc/chromebooks/preparing-chromebook.md:1]: https://github.com/hexdump0815/imagebuilder/pull/247/commits/82d48685b5625218ad07643043002470d2135661
here afaik only main readme pointed there
first-steps still points to installations step

File: doc/chromebooks/installation/basic-installation.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+├─mmcblk1p3  179:3    0   512M  0 part
+├─mmcblk1p4  179:4    0 107,9G  0 part
+└─mmcblk1p5  179:5    0     8G  0 part 
+mmcblk1boot0 179:32   0     4M  1 disk 
+mmcblk1boot1 179:64   0     4M  1 disk
+```
+
+_Tip. the chromebook partiotion will be the one without partition with "/"_
+
+_Note. the internal memory is often (but not always) ```mmcblk1``` or ```mmcblk0``` but_
```
[the remark this answers is not in the document]
Comment (thenameisluk) [reply doc/chromebooks/installation/basic-installation.md]: https://github.com/hexdump0815/imagebuilder/pull/247/commits/ab3c573539335598a10ac5c38d32d99fcaf67312

File: doc/chromebooks/installation/luks-installation.md [hunk 5]
```diff
@@ -48,6 +38,8 @@ we will export the correct drive as a environment variable but i still encourage
 
 _Note. remmber to do it whenever you want to repeat any of steps below_
 
+_Note. the internal memory is often (but not always) ```mmcblk1``` or ```mmcblk0``` but_
```
[the remark this answers is not in the document]
Comment (thenameisluk) [reply doc/chromebooks/installation/luks-installation.md]: https://github.com/hexdump0815/imagebuilder/pull/247/commits/ab3c573539335598a10ac5c38d32d99fcaf67312

File: doc/chromebooks/installation/luks-installation.md (hunk 2 above)
Comment (thenameisluk) [reply doc/chromebooks/installation/luks-installation.md:113]: https://github.com/hexdump0815/imagebuilder/pull/247/commits/5a844d546cfa9755c96cfc63d438b1b05d82c483

File: luk's special todo [hunk 6]
```diff
@@ -0,0 +1,33 @@
+will publicly track stuff for next pr
```
[the remark this answers is not in the document]
Comment (thenameisluk) [reply luk's special todo]: sure https://github.com/hexdump0815/imagebuilder/pull/247/commits/cc3a57082336a46ac85b44464f2c472db649fcd4

**Comment by thenameisluk** [comment]
@hexdump0815 moving docs to gitlab repo (at some point) might be a good idea it has 10gb (github 1gb) limit for pages and better build system for generating docs and stuff (like repos :3)

i've moved velvet repo there so i can host more stuff there

**Comment by thenameisluk** [comment]
i thing i got rid of all unreferenced png's

**Comment by thenameisluk** [comment]
seams like that's all requested changes

**Comment by hexdump0815** [comment]
re gitlab move: something to keep in mind maybe, but that is definitely for later - so far space there enough still and i prefer to keep things small if possible

**Pull request merged**